src/main/java/yanagishima/model/dto/HistoryStatusDto.java

```diff
@@ -0,0 +1,11 @@
+package yanagishima.model.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class HistoryStatusDto {
+  private String status;
```

Comment:
The coding style is different. (indent)

Reply:
It's intentionally. I will apply internal new style later.